### ISSUE TITLE
Specify bank account to use for payout?

### ISSUE THREAD

**Issue by andrewdavidcostello** [paypal/PayPal-PHP-SDK#211]
I haven't been able to find any information regarding the title so thought it best to try here.

When making a payout request is it possible to specify the bank account in which the payment should be funded from or is it always funded from the primary account linked to the Paypal account? (Therefore you would need a completely separate Paypal account if you also needed to fund the payment from a different bank account?)

Apologies if this isn't the right place to ask but I haven't had a straight answer on this from anyone I have contacted thus far.

**Comment by jaypatel512** [comment]
Hey @andrewdavidcostello ! No worries. I actually got your question, however, I personally may not have the answer for you right away, but I have already asked about this to Payouts Team who are developing these APIs. I will respond you back to you on this, as soon as possible. 

Thanks for your patience.

**Comment by jaypatel512** [comment]
Hey @andrewdavidcostello ! Just an update, I got the reply from one of the internal technical support, is that Payouts currently use PayPal balance to pay for all the payments in that Payout. however, you could be able to set your default payment source to a bank, and that should do it.

However, there is no api support yet, to give such an ability. Anyway, I am yet to receive a response from Payout Team Developers, but will update this as soon as I hear back from them. Thank you.

**Comment by rolfhaag** [comment]
Hi @andrewdavidcostello 

I'm the Payouts Product Owner.  At this time all Payouts sent must be funded from balance in the sending PayPal account.  In this regard, Payouts works just like Mass Payments.  You're foreshadowing a desired capability - funding your Payouts from a bank on file on the sending PayPal account.  We're looking into that for this year.

-Rolf

**Comment by andrewdavidcostello** [comment]
Thanks for the quick replies on this. Good to hear you wish to implement this functionality as it would really help when payment needs to come from different sources on a per client basis. For now we will have to go with a new account for each in this case.

**Comment by tbuser** [comment]
@rolfhaag Whats the status of this feature?  We're running into the same problem.  We want to use payouts api to pull funds from our bank account not the paypal balance.  It doesn't appear to be pulling from bank account set as primary, only from the balance in paypal.

**Comment by rolfhaag** [comment]
@tbuser and @jaypatel512 Thanks for the ping - but I'm no longer at PayPal.  And don't have visibility into the roadmap anymore.

**Comment by ppmtscory** [comment]
@tbuser I will check with the internal team to see if this is still on the roadmap and/or where it lands.  I'll update here once I hear back.

**Comment by ppmtscory** [comment]
The team is waiting on dependencies from other services before they can implement this feature.  Therefore, there isn't any ETA that I can provide with any degree of confidence.  I will, however, update here when/if I get any other information as to an ETA.

**Comment by sblawrie** [comment]
@ppmtscory @braebot Any update on this issue?  

I'm trying to automate payouts in my application and can't do it unless I login to the PayPal dashboard first and deposit enough funds from my bank account into the PayPal account.  Ideally, I could choose a source to make all of the payments from during the API call.

**Comment by jaypatel512** [comment]
Hey @sblawrie ! 

The feature is still in the pipeline, and the payouts team have not finished up on this task just yet. We will try and pass on the ask further to the internal team and see if we can influence the priority on this. We will keep you informed.

**Comment by sblawrie** [comment]
@jaypatel512 Much appreciated - thanks.

**Comment by ivpusic** [comment]
@jaypatel512 any updates? can we now run payout and if there is no money on paypal ballance it will be automatically taken from linked primary bank account?

thanks

**Comment by braebot** [comment]
This issue is now at https://github.com/paypal/PayPal-REST-API-Issues/issues/37. Follow this new issue to receive updates.
